tests/src/JIT/HardwareIntrinsics/X86/Shared/GenerateTests.csx

```diff
@@ -186,9 +203,6 @@ private static readonly (string templateFileName, Dictionary<string, string> tem
-    ("SimpleBinOpTest.template",    new Dictionary<string, string> { ["Isa"] = "Sse2", ["LoadIsa"] = "Sse2", ["Method"] = "PackSignedSaturate",                       ["RetVectorType"] = "Vector128", ["RetBaseType"] = "Int16",  ["Op1VectorType"] = "Vector128", ["Op1BaseType"] = "Int32",  ["Op2VectorType"] = "Vector128", ["Op2BaseType"] = "Int32",                                           ["LargestVectorSize"] = "16", ["NextValueOp1"] = "TestLibrary.Generator.GetInt32()",  ["NextValueOp2"] = "TestLibrary.Generator.GetInt32()",  ["ValidateFirstResult"] = "Sse2Verify.PackSignedSaturate(result, left, right)",                                                                                 ["ValidateRemainingResults"] = "Sse2Verify.PackSignedSaturate(result, left, right)"}),
-    ("SimpleBinOpTest.template",    new Dictionary<string, string> { ["Isa"] = "Sse2", ["LoadIsa"] = "Sse2", ["Method"] = "PackSignedSaturate",                       ["RetVectorType"] = "Vector128", ["RetBaseType"] = "SByte",  ["Op1VectorType"] = "Vector128", ["Op1BaseType"] = "Int16",  ["Op2VectorType"] = "Vector128", ["Op2BaseType"] = "Int16",                                           ["LargestVectorSize"] = "16", ["NextValueOp1"] = "TestLibrary.Generator.GetInt16()",  ["NextValueOp2"] = "TestLibrary.Generator.GetInt16()",  ["ValidateFirstResult"] = "Sse2Verify.PackSignedSaturate(result, left, right)",                                                                                 ["ValidateRemainingResults"] = "Sse2Verify.PackSignedSaturate(result, left, right)"}),
-    ("SimpleBinOpTest.template",    new Dictionary<string, string> { ["Isa"] = "Sse2", ["LoadIsa"] = "Sse2", ["Method"] = "PackUnsignedSaturate",                     ["RetVectorType"] = "Vector128", ["RetBaseType"] = "Byte",   ["Op1VectorType"] = "Vector128", ["Op1BaseType"] = "Int16",  ["Op2VectorType"] = "Vector128", ["Op2BaseType"] = "Int16",                                           ["LargestVectorSize"] = "16", ["NextValueOp1"] = "TestLibrary.Generator.GetInt16()",  ["NextValueOp2"] = "TestLibrary.Generator.GetInt16()",  ["ValidateFirstResult"] = "Sse2Verify.PackUnsignedSaturate(result, left, right)",                                                                               ["ValidateRemainingResults"] = "Sse2Verify.PackUnsignedSaturate(result, left, right)"}),
```

Comment:
It turns out that merge error was in previous PR #19673 already merged into master. Pack* and Unpack* tests were left in GenerateTests.csx file while all code generated by them, verification logic, and includes in Sse2_r.csproj, Sse2_ro.csproj projects and Program.Sse2.cs file were missing.

In this state code generated from GeneretaTests.csx will not compile as it requires additional changes to code which are included in PR #19670. To restore ability to generate and recompile tests using script I have had to remove all Pack* and Unpack* tests cases from this file.

Reply:
I would do this as a separate PR (or, at the very least as a separate commit).

It is very confusing to see "Convert Sse2 ConvertTo* and ConvertToVector128* tests to template based" and then look into the contents and see that it is deleting unrelated tests.

Reply:
Will correct in separate PR.